lib/api/controllers/routerController.js

```diff
@@ -153,13 +153,13 @@ module.exports = function RouterController (kuzzle) {
         {verb: 'delete', url: '/:index/:collection/:id', controller: 'write', action: 'delete'},
         {verb: 'delete', url: '/:index/:collection', controller: 'admin', action: 'deleteCollection'},
 
-        {verb: 'put', url: '/roles/:id', controller: 'security', action: 'putRole'},
-        {verb: 'put', url: '/profiles/:id', controller: 'security', action: 'putProfile'},
+        {verb: 'put', url: '/roles/:id', controller: 'security', action: 'createOrReplaceRole'},
+        {verb: 'put', url: '/profiles/:id', controller: 'security', action: 'createOrReplaceProfile'},
         {verb: 'put', url: '/:index', controller: 'admin', action: 'createIndex'},
         {verb: 'put', url: '/:index/:collection', controller: 'write', action: 'createCollection'},
-        {verb: 'put', url: '/:index/:collection/_mapping', controller: 'admin', action: 'putMapping'},
+        {verb: 'put', url: '/:index/:collection/_mapping', controller: 'admin', action: 'createOrReplaceMapping'},
```

Comment:
I have a doubt about the decision `createOrReplaceMapping`. Because we can't replace a mapping, just add some rules. Are you sure this is what we decided ?